### ISSUE TITLE
Improve model speed

### ISSUE THREAD

**Issue by FilipeTabosa** [experiencor/keras-yolo3#128]
Hey guys. I used this model to train a plate classifier and, by far, this code is the one that gave me the best results yet. But there is one simple problem. The code runs way TOO slow without a GPU compared to other solutions (like cascade for example). I even tried using tiny-yolo3, but it did not give me a good speed.
Do you guys have any idea how could I improve the speed of the code on CPU?
My first thought was about transforming it to tensorflow and compiling it with bazel; However, I cannot do that was all compilers found on internet asks for the architecture file. How could I obtain this file, and do you guys have any other ideas to improve this code speed (even if it means to lose a little of accuracy )

**Comment by NineBall9** [comment]
The same question! It costs 10 hours for me to train an epoch!!!

**Comment by TiongSun** [comment]
same here. We have big data, and it may take up to 2 years to finish training on Tesla K80...
